src/cloudtrail/README.md

```diff
@@ -1,18 +1,52 @@
-# Coralogix-CloudTrail
+# AWS cloud trail integarion for Coralogix
 
-This application retrieves **CloudTrail** logs and sends them to your **Coralogix** account.
+Coralogix provides a seamless integration with ``AWS`` cloud so you can send your logs from anywhere and parse them according to your needs.
```

Comment:
Coralogix provides a predefined Lambda function to easily forward your CloudTrail logs straight to the Coralogix platform.